controls/scheduler/built-in-dialogs.md

```diff
@@ -106,6 +106,28 @@ You can style the visual appearance of each dialog by creating the corresponding
 * `DeleteRecurrenceChoiceDialogStyle`&mdash;Defines the `Style` that will be applied on the Delete Recurrence Choice dialog.
 * `PreviewAppointmentDialogStyle`&mdash;(Android- and iOS-only)Defines the `Style` for the Preview Appointment dialog. 
 
+**Example with styling the edit appointment dialog**
+
+**1.** Define the Scheduler with the `EditAppointmentDialogStyle` in XAML:
+
+```XAML
+ <telerik:RadScheduler x:Name="scheduler"
+                       EditAppointmentDialogStyle="{StaticResource CustomSchedulerEditAppointmentDialog_Style}"/>
+```
+
+**2.** Define the style in the resources of the page:
+
+```XAML
+<Style x:Key="CustomSchedulerEditAppointmentDialog_Style" TargetType="telerik:SchedulerDialog">
+    <Setter Property="ControlTemplate" Value="{StaticResource CustomSchedulerEditAppointmentDialog_ControlTemplate}" />
+    <Setter Property="HeightRequest" Value="{OnPlatform Default=450, WinUI=530}" />
+</Style>
+```
+
+Review the `ControlTemplate` definition in the <a href="https://github.com/telerik/maui-samples/tree/main/Samples/ControlsSamples/Examples/SchedulerControl/CustomizationExample/Resources" target="_blank">Resources folder of the Controls Samples application > Scheduler/Customization example</a>.
```

Comment:
You provide a direct link, so you don't need to describe the location word by word as the sentence gets too confusing.
```suggestion
Review the `ControlTemplate` definition in the <a href="https://github.com/telerik/maui-samples/tree/main/Samples/ControlsSamples/Examples/SchedulerControl/CustomizationExample/Resources" target="_blank">`Resources` folder of the Controls Samples application.
```